src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumStarterRemoteProxy.java

```diff
@@ -389,7 +384,7 @@ boolean startDockerSeleniumContainer(String browser) {
                         .hostConfig(hostConfig)
                         .build();
 
-                String containerName = String.format("%s_%s", "zalenium", nodePort);
+                String containerName = String.format("%s_%s", getContainerName(), nodePort);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "containerName" which hides the field declared at line 83. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)